### PR TITLE
[fix] Correct upgrade script

### PR DESCRIPTION
**Pull request by zamentur**
Upgrade tested

### CODE REVIEW

**Comment by zamentur** [comment]
Fix https://github.com/YunoHost-Apps/mailman_ynh/issues/21 and https://github.com/YunoHost-Apps/mailman_ynh/issues/7

File: scripts/upgrade [hunk 1]
```diff
@@ -1,25 +1,188 @@
...
+chown -hR list:list /var/lib/mailman/
+/var/lib/mailman/bin/check_perms -f > /dev/null
+chown list:www-data -R /var/lib/mailman/archives/
+
+#=================================================
+# INTEGRATE SERVICE IN YUNOHOST
+#=================================================
+ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
+
+yunohost service add $app --description "A short description of the app" --log "/var/log/$app/$app.log"
```
Comment (alexAubin) [inline scripts/upgrade:149]: Description should be improved, + pretty sure that /var/log/mailman/mailman.log doesnt exist :s

File: scripts/upgrade [hunk 2]
```diff
@@ -1,25 +1,188 @@
...
+
 #=================================================
+# START SYSTEMD SERVICE
+#=================================================
+ynh_script_progression --message="Starting a systemd service..." --weight=1
+
+ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log"
```
Comment (alexAubin) [inline scripts/upgrade:156]: (Same here, we probably don't have a "/var/log/$app/$app.log")

File: scripts/upgrade [hunk 3]
```diff
@@ -1,25 +1,188 @@
...
+chown -hR list:list /var/lib/mailman/
+/var/lib/mailman/bin/check_perms -f > /dev/null
+chown list:www-data -R /var/lib/mailman/archives/
+
+#=================================================
+# INTEGRATE SERVICE IN YUNOHOST
+#=================================================
+ynh_script_progression --message="Integrating service in YunoHost..." --weight=1
+
+yunohost service add $app --description "Error log of mailman" --log "/var/log/$app/error"
```
Comment (alexAubin) [inline scripts/upgrade:149]: The description is supposed to describe the app, not the log ...

Also if we're putting "yunohost service add" in the upgrade script, it should be in the install and restore script as well

**Comment by ericgaspar** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**